sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -378,6 +378,8 @@ class AfterCount(TriggerFn):
   COUNT_TAG = _CombiningValueStateTag('count', combiners.CountCombineFn())
 
   def __init__(self, count):
+    if not isinstance(count, int) or count < 1:
```

Comment:
<!--thread_id:cc_158342029_t; commit:2df1e910f7e7508a6253afeb297a7487f5d5c392; resolved:0-->
<!--section:context-quote-->
> **aaltay** wrote:
> I meant a long value as in `type(3L) = <type 'long'>`. I do not see any reason for not letting users to pass long values here.

<!--section:body-->
It makes sense. I have just incorporated the right change so that long types are allowed while enforcing non-fractional numbers.